bin/release.sh

```diff
@@ -36,24 +50,17 @@ npx lerna run build \
   --ignore @bugsnag/plugin-electron-app \
   --ignore @bugsnag/plugin-electron-client-state-persistence
 
+# publish
+if [ -z RETRY_PUBLISH ]; then
+  npx lerna publish from-package
+else
+  npx lerna publish from-git
```

Comment:
Does this command push git tags to the remote? 

the [documentation](https://github.com/lerna/lerna/tree/main/libs/commands/publish#bump-from-git) suggests this is the command to use after versioning manually.

> This is useful in CI scenarios where you wish to manually increment versions, but have the package contents themselves consistently published by an automated process.